src/dataExplorer/components/DeleteScript.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {FC, useContext} from 'react'
+import {Button, ComponentColor, IconFont, Overlay} from '@influxdata/clockface'
+import './SaveAsScript.scss'
+import {CLOUD} from 'src/shared/constants'
+import {PersistanceContext} from 'src/dataExplorer/context/persistance'
+import {useHistory} from 'react-router-dom'
+import {useSelector, useDispatch} from 'react-redux'
+import {getOrg} from 'src/organizations/selectors'
+import {notify} from 'src/shared/actions/notifications'
+import {deleteScriptFail} from 'src/shared/copy/notifications/categories/scripts'
+
+let deleteScript
+
+if (CLOUD) {
+  deleteScript = require('src/client/scriptsRoutes').deleteScript
+}
+
+interface Props {
+  onBack: () => void
+  onClose: () => void
+}
+
+const OpenScript: FC<Props> = ({onBack, onClose}) => {
```

Comment:
```suggestion
const DeleteScript: FC<Props> = ({onBack, onClose}) => {
```
Should it be `DeleteScript` according to the file name?